src/Build/Evaluation/IntrinsicFunctions.cs

```diff
@@ -28,11 +29,11 @@ namespace Microsoft.Build.Evaluation
     /// </summary>
     internal static class IntrinsicFunctions
     {
-#if FEATURE_WIN32_REGISTRY
+#pragma warning disable CA1416 // Platform compatibility: we'll only use this on Windows
```

Comment:
Can you not put the platform guard on a property? Thought you could.

Reply:
Property yes, field no, so I suppressed the warning here rather than check the OS inside the set operation.

Reply:
Not sure why I thought that was a property last week 👍